examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,115 @@
+"""
+=========================================
+Receptive Field Estimation and Prediction
+=========================================
+
+This example re-creates figures from a paper from the Lalor group which
+demos the mTRF toolbox in Matlab [1]. We will show how the ReceptiveField
```

Comment:
we should standardize this for the other examples! Maybe I can do this at the docathon :)